lib/ufe/UsdObject3dHandler.cpp

```diff
@@ -32,6 +34,12 @@ Ufe::Object3d::Ptr UsdObject3dHandler::object3d(const Ufe::SceneItem::Ptr& item)
 #if !defined(NDEBUG)
 	assert(usdItem);
 #endif
+
+	// Test if this item is imageable. If not, then we cannot create an object3d
+	// interface for it, which is a valid case (such as for a material node type).
+	UsdGeomImageable primSchema(usdItem->prim());
+	if (!primSchema) return nullptr;
```

Comment:
Hardened this method by making sure it only returns interface for imageable items.